app/gui/view/component-browser/component-group/src/lib.rs

```diff
@@ -194,7 +221,8 @@ impl component::Frp<Model> for Frp {
                     model.update_header_width(*size, *hdr_geom);
                 })
             );
-            eval input.set_background_color((c) model.background.color.set(c.into()));
+            model.header.set_color_all <+ header_color;
```

Comment:
Again, I think "set_default_color" is better. If someone (for some weird reason) will add letters by editing API, the new letters will have default color. Perhaps even setting with `set_content` will fall back to default color as well.

Reply:
As previously, never leave code "as is". Adam is very right. Either change it to default color, or explain in detail why usig detailed color is impossible right now in a fixme here. Telling "its imposible" is not good enough. I also think it sohuld use default colors instead.